Documentation/ViewDesigner/Examples/VDTabbed.md

```diff
@@ -0,0 +1,381 @@
+## View Designer Tabbed - json
+
+To replicate the View Designer Tabbed Workbook, copy and paste the following code into the Advanced editor, denoted by the </> symbol in toolbar
+
+![Avanced Editor Toolbar](./ToolBar.png)
+
+Users may have to update their query settings and subscriptions to their own accessible resources
+
+[Return to Conversion Options](../ConversionOptions.md)
+
+```
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "/subscriptions/1f3fa6d2-851c-4a91-9087-1a050f3a9c38/resourcegroups/defaultresourcegroup-eus/providers/microsoft.operationalinsights/workspaces/defaultworkspace-1f3fa6d2-851c-4a91-9087-1a050f3a9c38-eus"
+        ],
+        "parameters": [
+          {
+            "id": "81018bf4-b214-4d2f-bfac-9efb30ea7afb",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "value": "/subscriptions/5c038d14-3833-463f-a492-de956f63f12a",

```

Comment:
strip this one out as well.
Do you have a personal MSDN subscription to try the steps out on?
We get one for free w/ $150 limit